<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gest_avion</class>
 <widget class="QDialog" name="gest_avion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>1311</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QFrame" name="FB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#FB{
border : 3px solid rgb(0, 255, 255);
border-radius : 40px;
	background-image: url(:/new/prefix1/airport-terminal.jpg);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="F1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>271</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#F1{
	background-color:transparent;
border : 3px solid rgb(0, 255, 255);
border-radius : 40px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Id_a_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Id_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Capacite_a_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;Capacite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Capacite_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Problem_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter_a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="Problem_a_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;Problem&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EC_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FC_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BC_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="BC_a_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;Business class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="FC_a_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;First class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="EC_a_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;Economy class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Charge_a_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Charge_a_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;Charge&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Modifier_a">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="F2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#F2{
border : 3px solid rgb(0, 255, 255);
border-radius : 40px;
background-color:transparent;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;avion ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="Fix_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>371</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{background:transparent} QTabWidget::pane{border: 1px;border-color:redt;background-color: transparent;} 
QTabBar::tab {background-color: transparent;color: rgb(170, 0, 255);width: 100px;height:20px;font-size:18px} 
QTabBar::tab:hover{background-color:  rgb(255, 151, 6); color: white;}
QTabBar::tab:selected{background-color:  rgb(7, 202, 255); color: white;}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>problem</string>
      </attribute>
      <widget class="QPushButton" name="Add_Problem_Button_a">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="Capacite_a_label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;ID Empolye&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="Id_a_label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;ID Problem&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Id_Problem_a">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Employe_combobox_a">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>fix</string>
      </attribute>
      <widget class="QComboBox" name="Problem_combobox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;problem deja exist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Fixed</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QPushButton" name="Image_a">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>157</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>ADD IMAGE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_img_a">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>301</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLineEdit" name="Id_a_input_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>113</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="F3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>441</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#F3{
border : 3px solid rgb(0, 255, 255);
background-color:transparent;
border-radius : 40px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>401</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{background:transparent} QTabWidget::pane{border: 1px;border-color:redt;background-color: transparent;} 
QTabBar::tab {background-color: transparent;color: rgb(170, 0, 255);width: 100px;height:20px;font-size:18px} 
QTabBar::tab:hover{background-color:  rgb(255, 151, 6); color: white;}
QTabBar::tab:selected{background-color:  rgb(7, 202, 255); color: white;}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>391</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Table</string>
      </attribute>
      <widget class="QTableView" name="tableAvion">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>401</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="search_a_input">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 3px solid #363525;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="Id_a_label_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Tri_Avion">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Tri</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Tri_Combobox_a">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>22</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Choix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ID_AVION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CAPACITE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FIRST_CLASS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BUSINESS_CLASS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ECONOMY_CLASS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CHARGE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="Supprimer_a">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>230</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>avion hors service permanant</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF_a">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; color:#ff0000;&quot;&gt;Gestion des avions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 203, 0);
border-width: 1px;
border-color: none;
border-style: solid;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <zorder>F3</zorder>
   <zorder>F1</zorder>
   <zorder>F2</zorder>
   <zorder>label_3</zorder>
   <zorder>back</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
